config/routes.rb

```diff
@@ -1,4 +1,4 @@
 Rails.application.routes.draw do
-  root to: "application#index"
+  mount_ember_app :exam_builder, to: "/", controller: "application", action: "index"
```

Comment:
* `root to: "application#index"` doesn't properly hook into the EmberCLI
  ecosystem. Instead, use `mount_ember_app`:

```ruby
mount_ember_app :exam_builder,
  to: "/",
  controller: "application",
  action: "index"
```